release/release.go

```diff
@@ -151,6 +151,62 @@ func CheckUpstreamRelease(ctx context.Context, client *github.Client, org, repo
 	return releases, nil
 }
 
+func KubernetesGoVersion(ctx context.Context, client *github.Client) (map[string]string, error) {
```

Comment:
This function should take the version of Kubernets we're looking for. ie `v1.24.13`

Reply:
Done, I changed this function as expected